.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+# DDCE-StoryNumberHere
```

Comment:
do we really need, this, i never seen people updating it?

Reply:
I'd say it's worth keeping just for new joiners to use